rectools/models/sasrec.py

```diff
@@ -526,6 +512,25 @@ def transform_dataset_i2i(self, dataset: Dataset) -> Dataset:
 class SASRecDataPreparator(SessionEncoderDataPreparatorBase):
     """TODO"""
 
+    def __init__(
+        self,
+        session_max_len: int,
+        batch_size: int,
+        dataloader_num_workers: int,
+        n_negative: int,
```

Comment:
n_negatives here and everywhere for consistency with other `n_` uses like `n_factors`

Reply:
fixed